plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -176,7 +208,10 @@ const MainContent: React.FC< MainContentProps > = ( { setHasError } ) => {
 				<TermsOfService />
 			</p>
 
-			<GetStartedButton setHasError={ setHasError } />
+			<div className="wcpay-empty-subscriptions__button_container">
+				<NoThanksButton></NoThanksButton>
```

Comment:
**Nitpick:** This could be converted into a [self-closing tag](https://react-cn.github.io/react/tips/self-closing-tag.html), e.g. `<NoThanksButton />`

Reply:
Fixed in [59478e3](https://github.com/woocommerce/woocommerce/pull/32913/commits/59478e391c51cdc38adf12cae80988a019126c40)